cpp/box/Box.h

```diff
@@ -381,6 +382,53 @@ class Box
         });
     }
 
+    //! Calculate distance between two points using boundary conditions
+    /*! \param p_i Particle position
+        \param p_j Particle position to calculate distances from
+    */
+    float computeDistance(const vec3<float>& p_i, const vec3<float>& p_j) const
+    {
+            vec3<float> v_ij = wrap(p_j - p_i);
+            return std::sqrt(dot(v_ij, v_ij));
+    }
+
+    //! Calculate distance between a set of points and query points using boundary conditions
+    /*! \param points Particle positions
+        \param query_points Particle position to calculate distances from
+        \param dist Distances between points and query_points
+        \param Nvecs The number of points to calculate distances between
+    */
+    void computeDistances(vec3<float>* points, vec3<float>* query_points, float *dist, unsigned int Nvecs) const
```

Comment:
I think I want to see `query_points` listed before `points`, everywhere in this pull request. This follows the convention that bonds point from query points to points. I would like @vyasr to confirm my preference before you rewrite all of the code, however.

Reply:
I'm fine with this change.